featuretests/cmd_juju_current_controller_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package featuretests
+
+import (
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/cmd"
+	"github.com/juju/cmd/cmdtesting"
+	"github.com/juju/loggo"
+	jc "github.com/juju/testing/checkers"
+
+	"github.com/juju/juju/cmd/juju/commands"
+	jujutesting "github.com/juju/juju/juju/testing"
+	"github.com/juju/juju/jujuclient"
+)
+
+type cmdCurrentControllerSuite struct {
+	jujutesting.JujuConnSuite
```

Comment:
Is this correct?

Reply:
yes. it's in featuretest pkg for a reason :)